pandas/core/generic.py

```diff
@@ -67,7 +67,8 @@
     args_transpose='axes to permute (int or label for object)',
     optional_by="""
         by : str or list of str
-            Name or list of names to sort by""")
+            Name or list of names to sort by
```

Comment:
why are you changing all of this?

Reply:
Apologies...That was in response to the validate. Let me push jus tthe mean additions from my local.